hexa/workspaces/schema/mutations.py

```diff
@@ -294,7 +298,25 @@ def resolve_resend_workspace_invitation(_, info, **kwargs):
         invitation.updated_at = datetime.utcnow()
         invitation.save()
 
-        send_workspace_invitation_email(invitation)
+        query_set = User.objects.filter(email=invitation.email)
```

Comment:
This can be in `send_workspace_invitation_email` to not have two cases here. 
Plus, I'd prefer to have a `user = User.objects.filter(email=invitation.email).first()` than this, it's harder to read.